software/wmla120_ansible/engr_mode_env_pre_install_gather.yml

```diff
@@ -86,39 +87,40 @@
 
 - name: Check if dlipy2 pre data exists (PIP)
   stat:
-    path: "~/{{ dlipy2_pip }}"
+    path: "{{ ansible_env.HOME }}/{{dlipy2_pip}}"
+>>>>>>> 4a16ee2258c6900f005fbf3b751a4a304a739bb3
```

Comment:
hmm ... have you unit tested this code after git merge? Seems that this is incorrect.

Reply:
Yes..just needed cleanup due to resolved conflicts.